src/style.css

```diff
@@ -0,0 +1,104 @@
+@font-face {
+  font-family: Avenir;
+  src: url(/fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  outline: none;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+}
+
+.input {
+  width: 100%;
+  margin-top: 20px;
+  margin-bottom: 20px;
+  font-family: Avenir, sans-serif;
+  box-sizing: border-box;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  color: #3d4e61;
+  display: block;
+}
+
+.input--big {
+  height: 70px;
+  font-size: 16px;
+  line-height: 22px;
+  padding-left: 62px;
+}
+
+.input--small {
+  height: 42px;
+  font-size: 14px;
+  line-height: 19px;
+  padding-left: 33px;
+}
+
+.big-search-form {
+  position: relative;
+  margin-top: 20px;
+}
+
+.small-search-form {
+  position: relative;
+  margin-top: 20px;
+}
+
+.image-for-big-search {
+  position: absolute;
+  top: 25px;
+  left: 26px;
+  width: 19px;
+  height: 19px;
+}
+
+.image-for-small-form {
+  position: absolute;
+  top: 15px;
+  left: 13px;
+  width: 11px;
+  height: 11px;
+}
+
+.input:default {
+  box-sizing: border-box;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.input:hover {
+  box-sizing: border-box;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.input:focus {
+  box-sizing: border-box;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  font-weight: 900;
+}
+
+.input--big::placeholder {
+  font-weight: 300;
+  color: #3d4e61;
+  position: absolute;
+  left: 62px;
+  top: 25px;
+}
+
+.input--small::placeholder {
+  font-weight: 300;
+  color: #3d4e61;
+  position: absolute;
+  left: 33px;
+  top: 12px;
+}
```

Comment:
take a look over 2 point of the [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) and fix it
also fix all other similar pairs of selectors
![Screenshot 2023-02-02 at 10 04 48](https://user-images.githubusercontent.com/94289038/216266406-c981d5f6-2b07-49f1-9ea4-2932c332dbd1.png)

do not repeat yourself